classes/class5/README.md

```diff
@@ -1 +1,124 @@
+
+CSS also supports the absolute units `cm`, `mm`, `in`, `pt`, and `pc`, but these are better suited for print than the screen.
+
+### Selectors and Specificity
+
+The most common selector is a `tagname`, a `class` (preceded by `.`), or an `id` (preceded by `#`), though there are many other ways to select elements to style.
+
+Styles of elements _cascade_ (apply) to their child elements. In cases where multiple conflicting styles are applied to an element, _the more specific style prevails_.
+
+So with this document:
+
+```css
+<div class="styled">Content</div>
+```
+
+and this style:
+
+```css
+div {
+  color white;
+  background-color: red;
+}
+
+.styled {
+  background-color: blue;
+}
+```
+
+The single `div` in the document will have a blue background color because the selector `.styled` is more specific than `div`. Paste the above into [Codepen](http://codepen.io/pen/) to test it out.
+
+### Colors
+
+A quick aside on colors, which can be specified in CSS in multiple ways.
+
+| Method | Description |
+|----|----
+| `#RRGGBB` | Red, green, and blue are each specified by a two-digit hex number, `00` to `FF`. White is `#FFFFFF` and black is `#000000`.
+| `#RGB` | Same as above but with half the precision.
+| `name` | Certain colors (like red, blue, yellow, etc.) can be specified simply by name. They're mostly hideous but black and white are fine.
+| `rgb(r,g,b)` | Red, green, and blue are specified on a 0-255 scale.
+| `rgba(r,g,b,a)` | Same as above, but alpha (opacity) is specified on a 0-1 scale.
+
+Here's a neat [clock](http://whatcolourisit.scn9a.org) that displays the color specified by the hex code corresponding to the current time.
+
+### Box Model
+
+All HTML elements can be modeled as boxes. CSS allows us to modify that box to affect the display of content. Below is the styling for a box and its corresponding box model. The properties under `/* metrics */` determine the size of the box and the spacing of its contents.
+
+```css
+.styled {
+  background-color: blue;
+  color: white;
+  text-align: center;
+  
+  /* metrics */
+  margin: 8px;
+  border: 2px dotted black;
+  padding: 8px;
+  width: 100px;
+}
+```
+
+![ Box model ]( box.png )
+
+In the image above, the solid border around the yellow box defines the outer edge of the element. The `margin` is space between the box and its surroundings. The `border` is space between the outer edge and the `padding`, which defines the space between the inner edge of the border and the content of the element.
+
+As the image above shows, the `width` property sets the width of the *content*, not the width of the entire element. This means that `padding` and `border` add to the apparent dimensions of the box. We can add the below property to change that:
+
+```css
+box-sizing: border-box;
+```
+
+With this property set, the `width` and `height` properties set the dimensions of the box, and those dimensions are not affected by `padding` or `border`.
```

Comment:
Perhaps useful to mention benefits either way of these models (or maybe something we just mention in class)